assgnment2/user.dart

```diff
@@ -0,0 +1,40 @@
+class User{
+  
+  String userName;
```

Comment:
Use `final / const / late` keyword according to your use case when declaring variables.